server/src/main/java/io/spine/server/delivery/DeliveryMonitor.java

```diff
@@ -95,6 +95,34 @@ public void onDeliveryStarted(ShardIndex index) {
         // do nothing.
     }
 
+    /**
+     * Called if an {@code Exception} occurred when the {@code Delivery} attempted
+     * to pick up a shard.
+     *
+     * <p>Pay attention, this method is not called if the shard could not be picked up because
+     * it's already picked by another worker. This situation doesn't lead to an exception in
+     * {@code ShardedWorkRegistry} so it is not considered as a failure.
+     *
+     * @param index
+     *         an index of the shard that could not be picked up due to an exception
+     */
+    @SuppressWarnings({"WeakerAccess", "unused"})  // This SPI method is designed for descendants.
+    public void onShardPickUpFailure(ShardIndex index) {
+        // do nothing.
```

Comment:
Could you please change this (and every other case in this file to a capitalized version:
```java
// Do nothing.
```